### DISCUSSION TITLE
Logo for Swiss?

### DISCUSSION THREAD

**Discussion by coolaj86** [emukidid/swiss-gc#509] [Ideas]
If I do a Google image search for "GC Swiss Logo", these are the kinds of things that pop up.

Is there an official logo?

| ![SwissLogo square](https://user-images.githubusercontent.com/122831/109929433-ab36fe00-7c83-11eb-8026-53429536d873.png)  | ![Swiss Homebrew Disc](https://user-images.githubusercontent.com/122831/109929993-5a73d500-7c84-11eb-859d-40cb11dcf867.jpg)  |
| --- | --- |
| ![Swiss GameCube Memory Card Logo](https://user-images.githubusercontent.com/122831/109929706-01a43c80-7c84-11eb-9b01-8baba6165ffa.jpg)  |  ![Swiss Wii Menu](https://user-images.githubusercontent.com/122831/109929555-d3266180-7c83-11eb-9e3e-8f2341e71818.png) |

**Comment by emukidid** [comment]
There is not.

**Comment by coolaj86** [comment]
@emukidid Would you be open to having a logo?

I'd be happy to engage someone on a service like Fiverr and get some concepts for your approval.

Here are a few ideas I have for a possible logo
- have a GameCube logo stylized `S` or `+`
    - (keep the most popularly used style, without infringing the NGC trademark)
- `swiss cheese-ify` an image of a GameCube, similar to the one used as the swiss menu background image
- a menu-like icon that has a similar shape to the menu that currently loads, perhaps with the word "swiss" underneath

(or if you have any preferences or ideas, I'd love to hear about them)

I'm working on another GameCube Homebrew video right now (**update**: it's out: <https://youtu.be/lUTk3ARrltA>) and since Swiss is at the core of anything about Homebrew I'd love to have an official, approved way to represent it visually.

**Comment by emukidid** [comment]
I'm open to ideas and if it's something that's commissioned I'd be happy to contribute as well :)

As for my own personal opinion, the second option sounds like the one I'm leaning most towards, almost like swiss cheese but in a GameCube indigo colour and somehow reminiscent of a GameCube console.  I feel like this would work well as a logo in a banner but also in a larger background style.

Slightly off topic: A long time ago I had ambitions to make the Swiss menu be exactly like the GameCube IPL, e.g. a bouncing Swiss cheese-ified cube that would navigate you around the various areas in Swiss, but I don't think that'll eventuate as such. It'd probably be possible to make it an optional background though without too much work.

**Comment by coolaj86** [comment]
There's two people I've used on Fiverr before for another project that I think can come up with some good concepts for this. Last time I went in with a fairly clear idea of what I wanted and didn't want, but the guy spun off the idea I didn't like in a new way, and I was sold.

I'm going to bring both of those ideas to the table and see what they come up with.

## Mood Board

Here's my "mood board" of what I think we're going for with the swiss cheese style:

| ![backdrop](https://user-images.githubusercontent.com/122831/110073160-278d1800-7d3c-11eb-830e-3e85b70be4b9.png) | ![swiss-r4-backdrop](https://user-images.githubusercontent.com/122831/110073091-117f5780-7d3c-11eb-8f46-7575ff864c83.png) |
| --- | --- |
|  ![](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fcubemedia.ign.com%2Fmedia%2Fnews%2Fimage%2Ffaq%2Fgamecubeconsole.jpg&f=1&nofb=1) | ![](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fstatic.giantbomb.com%2Fuploads%2Fscale_medium%2F1%2F12482%2F822648-gamecube_1_1.jpg&f=1&nofb=1) |
| ![](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fmedia.gamestop.com%2Fi%2Fgamestop%2F10131396%2FNintendo-GameCube-System---Indigo%3F%24pdp%24&f=1&nofb=1) | ![](https://external-content.duckduckgo.com/iu/?u=http%3A%2F%2Fwww.seriouseats.com%2Fimages%2F2014%2F06%2F20140623-cheese101-hard-cheese-emmenthaler-vicky-wasik-1.jpg&f=1&nofb=1) | 
| ![](https://external-content.duckduckgo.com/iu/?u=http%3A%2F%2F3.bp.blogspot.com%2F-uTGwDPQxUeI%2FTwZS795vA4I%2FAAAAAAAABrk%2FRpD1asqHPOo%2Fs1600%2FSwissCheese.jpg&f=1&nofb=1) | ![](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse1.mm.bing.net%2Fth%3Fid%3DOIP.GWuBMsBfLn26G-a7EID98wHaDt%26pid%3DApi&f=1) |
| ![](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fwww.ecosystemmarketplace.com%2Fwp-content%2Fuploads%2F2019%2F11%2FSwiss-Cheese.jpg&f=1&nofb=1) | ![](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fwww.my7daydiet.com%2Fimages%2Fswiss-cheese.jpg&f=1&nofb=1) |
| ![NGC Logo Squarish](https://user-images.githubusercontent.com/122831/110081348-0a127b00-7d49-11eb-9382-9bb0509c63fc.jpg) | ![](https://images-wixmp-ed30a86b8c4ca887773594c2.wixmp.com/i/95f7b873-60b7-4cf6-a417-2f1350d72463/d95nsv3-b504c51d-a75c-42ed-b40e-dc22bc192c1d.jpg/v1/fill/w_894,h_894,q_75,strp/gamecube_startup_boot_animation_by_zygomatic_fool-d95nsv3.jpg?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJ1cm46YXBwOjdlMGQxODg5ODIyNjQzNzNhNWYwZDQxNWVhMGQyNmUwIiwic3ViIjoidXJuOmFwcDo3ZTBkMTg4OTgyMjY0MzczYTVmMGQ0MTVlYTBkMjZlMCIsImF1ZCI6WyJ1cm46c2VydmljZTppbWFnZS5vcGVyYXRpb25zIl0sIm9iaiI6W1t7InBhdGgiOiIvaS85NWY3Yjg3My02MGI3LTRjZjYtYTQxNy0yZjEzNTBkNzI0NjMvZDk1bnN2My1iNTA0YzUxZC1hNzVjLTQyZWQtYjQwZS1kYzIyYmMxOTJjMWQuanBnIiwid2lkdGgiOiI8PTg5NCIsImhlaWdodCI6Ijw9ODk0In1dXX0.wQM2CFifs1M0JSsEjrc8ImcJh62DNf5cYeAgM0Nb5LQ) |

![GameCube Cube](https://user-images.githubusercontent.com/122831/110420641-f6bd2380-8058-11eb-8bcc-b9157654d2cf.png)

## Rough Concepts / Experiments

If the GameCube were a block of Swiss Cheese you would be able to see four main cutouts on the top of a gamecube, as well as an additional 4 from the front.

They probably wouldn't actually extend the whole way through the block of cheese, but there may be some intersection for a few of them.

![IMG_9244](https://user-images.githubusercontent.com/122831/110194986-dc850a80-7df8-11eb-83be-899ce6de5f25.jpg)

The large circle with the GameCube logo could be replaced with S for Swiss, as if on the wrapper of the cheese.

![IMG_9245](https://user-images.githubusercontent.com/122831/110195624-ac8b3680-7dfb-11eb-9f33-3a569308c558.JPG)

Far too literal to my napkin sketch, but...

![GC Swiss Logo Draft A + B](https://user-images.githubusercontent.com/122831/110220692-92e20180-7e84-11eb-8c27-9b88a6216a5f.png)

**Comment by emukidid** [comment]
nice, thanks for doing this

**Comment by coolaj86** [comment]
Don't thank me yet. ;)

The person who I think would have nailed the style is no longer on Fiverr. :-/

Do any of these styles resonate with you?

| | A | B |
| -- | -- | -- |
| 1 | ![](https://fiverr-res.cloudinary.com/images/t_main1,q_auto,f_auto,q_auto,f_auto/gigs/190305653/original/eba15a44da71a021030566ac30fd4b0653dbd9e2/do-modern-unique-minimalist-business-logo-design.jpg) | ![](https://fiverr-res.cloudinary.com/images/t_main1,q_auto,f_auto,q_auto,f_auto/gigs2/114314925/original/2ea30bc4cadc2de2b193acb5f7b2250a998c8459/design-versatile-and-minimalist-3d-vintage-real-estate-logo.jpg) |
| 2 | ![](https://fiverr-res.cloudinary.com/images/t_main1,q_auto,f_auto,q_auto,f_auto/gigs/135588087/original/835df8494bb62f9de3849ab3207eb51e464dd07b/create-playful-game-title-logo-designs.png) | ![](https://fiverr-res.cloudinary.com/images/t_main1,q_auto,f_auto,q_auto,f_auto/gigs/123518981/original/6b57c2c591ff0513ab3ca5bedaba7107114ff831/design-a-professional-signature-logo-in-24hrs.jpg) |
| 3 | ![](https://fiverr-res.cloudinary.com/images/t_main1,q_auto,f_auto,q_auto,f_auto/gigs3/186924829/original/7f590c82b8ee8c65d1080622a6c704bd0bbd98c4/craft-a-vector-3d-logo-with-source-file-a6a5.jpg) | ![](https://fiverr-res.cloudinary.com/images/t_smartwm/t_main1,q_auto,f_auto,q_auto,f_auto/attachments/delivery/asset/434072f997c836199d2e061d43496a55-1614466591/viral%20leash-08/craft-a-vector-3d-logo-with-source-file-a6a5.png) |
| 4 | ![](https://fiverr-res.cloudinary.com/images/t_main1,q_auto,f_auto,q_auto,f_auto/gigs2/126765900/original/289c2a4b151500a38a47981ea0b1550666f99857/design-game-logo-for-you.png) | ![](https://fiverr-res.cloudinary.com/images/t_main1,q_auto,f_auto,q_auto,f_auto/attachments/delivery/asset/6d8a64a73a991f5a92c345af633e1852-1581023445/2%20-%20Logo%20Creo%20-%20GOD%20SIMULATOR%20--02/design-game-logo-for-you.png) |
| 5  | ![](https://mcusercontent.com/47ae772b45adba90321a881dd/images/442cb290-7a31-4e59-a409-ec7a11cab810.jpg) | ![](https://fiverr-res.cloudinary.com/images/t_smartwm/t_main1,q_auto,f_auto,q_auto,f_auto/attachments/delivery/asset/ec1252cc44111c7d138b80a360c17ef4-1605202675/BlinkPhace/design-creative-minimalist-logo.jpg) |
| 6 |  ![](https://fiverr-res.cloudinary.com/images/t_main1,q_auto,f_auto,q_auto,f_auto/gigs/146598170/original/2361aa8c364a8005003f52b0320aeb454a63781a/create-a-minimalist-pixel-art-logo-design.png) | ![](https://fiverr-res.cloudinary.com/images/t_main1,q_auto,f_auto,q_auto,f_auto/gigs/156806781/original/25342582bd9a537d16258aa96a5ad3349d2a1766/create-pixel-art-characters-or-items-for-your-project.png) |
| 7 | ![](https://fiverr-res.cloudinary.com/images/t_main1,q_auto,f_auto,q_auto,f_auto/gigs/153851985/original/371d7fc88248dae3282f04afc1ce7f35d5456cc2/create-a-vectorial-icons-and-cliparts.png) | ![](https://fiverr-res.cloudinary.com/images/t_smartwm/t_main1,q_auto,f_auto,q_auto,f_auto/attachments/delivery/asset/7fff73ac96ed1ba824458bdb4a0b621f-1614288512/2.25.21.006/draw-and-mail-caricature-colored-in-watercolors.jpg) |

**Comment by coolaj86** [comment]
@emukidid: Any thoughts on those styles of art?

Or the concepts I added to the mood board?

**Comment by coolaj86** [comment]
Feedback on these?

## Concepts

The GameCube as a block of cheese. Not very good. It's interesting take, but would need to be cleaned up a lot to make sense:

![GameCube Swiss Cube small](https://user-images.githubusercontent.com/122831/110503805-cc4f8280-80b9-11eb-84af-fda752c4f167.jpeg)

Looking at the S as a logo it's a little too flat, but I think it has promise:

![GameCube Swiss S Logo small](https://user-images.githubusercontent.com/122831/110504143-1a648600-80ba-11eb-92eb-c5823c05d9d7.png)

**Comment by coolaj86** [comment]
I think that the concept of "a slice or block of Swiss Cheese that looks like a gamecube" is not getting through.

I don't think these people know what a GameCube is, and I don't think their English is super great, on average - at least not at the $15 price point.

At $100 we could probably find someone who is fluent in English and does a great job. I paid about $250 in total (a mix of $5-$25 and $75-$100 gigs) to arrive at a logo earlier this year, and the end result was perfect, but I don't have enough skin in the game or decision power on this to invest that much.

**Comment by coolaj86** [comment]
@emukidid, Sorry to bombard with messages today, but I think this one turned out _really_ well.

I've got 24 hours to give feedback. Please take a few minutes to give me any direction if you can.

![GameCube Swiss Block small](https://user-images.githubusercontent.com/122831/110533477-45130680-80db-11eb-9f4f-fc4f239ab00a.png)

**Comment by 9ary** [comment]
I think it might be a good idea to simplify this further. Don't try to make it look too much like a GameCube, instead start with a cube of cheese and add a simple, but easily recognizable design feature of the console (the carry handle comes to mind).
It also doesn't necessarily have to be indigo/purple; for example, I think the natural yellow color of swiss cheese would work great. Or maybe try the glassy appearance of the animated cube from the IPL menu (not the startup animation).

**Comment by emukidid** [comment]
I'm on the same page as @9ary, it'd be nice if it looked more like the cube in the IPL menu (glassy), but I'm unsure what that would mean for the holes as they'd potentially need to depict depth somehow?

Specific feedback for the one posted [here](https://github.com/emukidid/swiss-gc/issues/489#issuecomment-794414282):
* Remove the S from the jewel
* Incorporate the carry handle

**Comment by coolaj86** [comment]
@9ary, @emukidid: I'm going to respond here real quick, but could we perhaps get on a voice chat to get talk through some ideas and get a clear vision?

So this is what we mean by "IPL Cube", right? (you had mentioned IPL earlier, but I actually didn't know what that meant)

![GameCube System (IPL) Menu small](https://user-images.githubusercontent.com/122831/110553971-1f93f600-80f7-11eb-949d-d69ef0b6b781.png)

II think that trying a concept with that would take some additional budget, and I think I'd need to re-review the designers to screen for someone that seems qualified to that art style.

With the budget I've allocated for this for myself ($50), I can get another revision in, but not pick a different designer.

@emukidid You said you'd be willing to contribute to the commission. What are you thinking?

**Comment by coolaj86** [comment]
Also, @emukidid, @9ary I think I understand what you mean with the block of cheese and handle, but could you do a quick napkin sketch to help me understand?

Are you thinking something like this?

![GameCube Swiss Yellow Chese block small](https://user-images.githubusercontent.com/122831/110554764-a7c6cb00-80f8-11eb-88bb-e3feb4396f82.png)

Or more at a perspective angle?

I'm not clear at all on how that might apply to the glassy look - and although that wouldn't come across on a napkin in terms of color, I could at least understand the perspective / shape / dimension.

**Comment by coolaj86** [comment]
Here's the updated GameCube:

![GameCube Swiss Orange on Black small](https://user-images.githubusercontent.com/122831/110673272-ce831100-818d-11eb-8b1d-f3454fe24fd1.png)

I've asked him to try an Orange-to-Yellow gradient rather than an Orange-to-White.

Due to the perspective, I'm not sure that the handle will make sense. For example, here's a scale-model gamecube at about that angle:

![GameCube perspective with handle](https://user-images.githubusercontent.com/122831/110674055-bd86cf80-818e-11eb-81b9-01720ac06e44.png)

Honestly, I like the Indigo better.

**Comment by 9ary** [comment]
I really don't like the aligned holes for the controller ports, they don't work with the cheese idea. Besides that I think it looks good.
The color should be more cheese-like.
The carry handle isn't critical considering the top design but it'd be nice to see what it could look like.
I'd also like to see more typography options for the text.

**Comment by loociano** [comment]
Sorry if this sounds silly, but isn't Swiss called after a [Swiss Army knife](https://en.wikipedia.org/wiki/Swiss_Army_knife) as opposed to some (delicious) cheese?

**Comment by 9ary** [comment]
Yes, but it's more fun to go for the cheese. :P

**Comment by ohnozen** [comment]
Hi, graphic designer here. I know I'm giving unsolicited opinions so feel free to ignore.

I love the memory card with the Swiss design aesthetic :)

What if the swiss cheese-ing created an S shape in the GameCube? There was also a drawing further up that laid the 'Cube on the side, could make a large hole in the middle for an S and reincorporate the handle.

Two 'Cubes laid back-to-back could also create an S shape with the handles. I see this in my mind palace as more of a minimal design, but that's also my style...

I don't know how deeply invested things are with Fiiver, but I'd be happy to design some mock-ups pro bono. Again feel free to ignore, just barging in rudely.

**Comment by emukidid** [comment]
A few days have passed and after looking at the mockups so far here I'm keen to move away from the cheese oriented design. I've contacted drmr from homebrew channel/Wii64 design fame and he's going to get back to me with some ideas as well. I'll provide updates as we go.

Always open to ideas so feel free to post anything you might have here.

**Comment by ohnozen** [comment]
Here's two first draft concepts. On the left is the S using the lid, right is using the handle to create the S form. Bottom designs stop overlapping at the bottom of the power button.

I'm partial to the bottom design, even if it is a wider design. After exporting I noticed the bottom design also works as an infinity loop. Happenstance playing out I guess.

I don't think the transparent version would be a good final design, but it does show how the overlap is working.

The colors feel off, but it might have been the photos I was using as reference...

Thoughts, changes, and criticisms are all welcome.

![swiss-01](https://user-images.githubusercontent.com/14218478/111344852-bfe3a080-864a-11eb-9af0-5a3d20d3985c.jpg)

**Comment by coolaj86** [comment]
What I like about the bottom row (symmetric) is that it'll scale well down to favicon and still be recognizable - and I like the S of the handles.

However, I think to match the time period / ecosystem it would fit better to have something with a little more dimension or texture.

Re: Fiverr cc/ @emukidid: I take no offense if you go a completely different route. Sunk costs on iteration are simply a part of the creative process. I missed my window for being able to give any more feedback, so if you decide later that you'd like more iterations on those designs I'd be happy to help get that going again - with the understanding that my budget is spent.

**Comment by ohnozen** [comment]
Properly rotated the cubes instead of just mirroring them, and started the decal adding process. I think this easier to do in an iterative process as designs can start looking busy *too fast*.

![swiss-02](https://user-images.githubusercontent.com/14218478/111378113-70fc3200-866f-11eb-8025-b1a520c7cba4.jpg)

**Comment by 9ary** [comment]
This looks good. I think rotating it a little to make the S properly vertical makes it even better (this is 30 degrees counter clockwise).
![image](https://user-images.githubusercontent.com/1155030/111378985-45973880-86ab-11eb-88af-4def69839770.png)

**Comment by ohnozen** [comment]
@9ary the 30deg is growing on me.

Just to be thorough: OG rotation, a "flat" version as a palette cleanser, and the 30deg version:

![swiss-03](https://user-images.githubusercontent.com/14218478/111384186-339ba280-8677-11eb-962b-b4078bd6042a.jpg)

**Comment by 9ary** [comment]
Yep, I also tried "flat" initially, but it just looks odd. As it turns out, "perfect" alignment/proportions/rotation doesn't always look good. :)

**Comment by ohnozen** [comment]
@9ary Yeeeeaaah I get too hung up on symmetry sometimes. Need to break out of that shell, so I appreciate you giving the design that nudge.

I'm stretching my legs a bit with the design: trying something monochrome, little tighter, and abstracting the Cube a tiny bit.

These are more of "getting it out of my system," but who knows? Maybe they *are* worth considering.

![swiss_u1](https://user-images.githubusercontent.com/14218478/111403540-22648d00-869b-11eb-8329-529f8ddae37f.png)

![swiss_u2](https://user-images.githubusercontent.com/14218478/111473793-6a1a0180-86f9-11eb-8d83-2de10281ad06.jpg)

**Comment by coolaj86** [comment]
@emukidid Any update from the designer of Wii fame?

(P.S. the ["Complete GameCube Game Save Exploits Guide"](https://youtu.be/lUTk3ARrltA) video I was working on is now out: <https://youtu.be/lUTk3ARrltA>)

**Comment by Kniteschaed** [comment]
A few ideas:

![1789883983759~2.jpg](https://github.com/user-attachments/assets/88c43bdf-9aad-42c0-b06b-112c1317fcab)

![1789884082555~2.jpg](https://github.com/user-attachments/assets/12224367-a3fa-4dde-8e32-dd2985303f38)

![1789884539379~2.jpg](https://github.com/user-attachments/assets/599b9837-034c-4837-bd47-198f584c93a0)

**Reply by JucaTollabur** [reply]
Ficou excelente esse com o queijo suíço.
